Game/ConsoleApplication2/AIManager.cpp

```diff
@@ -0,0 +1,14 @@
+#include "stdafx.h"
+#include "AIManager.h"
+
+//AIManager* AIManager::m_instance = 0;
```

Comment:
Never leave commented code in a PR, its either necessary for the code to run or not. If we want to make it a singleton later we will, but if we never do this code is clutter